include/wx/charts/wxmath2dplot.h

```diff
@@ -59,11 +59,12 @@ class wxMath2DPlotDataset
     /// @param dotStrokeColor The color of the pen
     /// used to draw the outline of the points.
     /// @param data The list of values.
+    /// @param showDots Enable displaz of dots on lines.
     wxMath2DPlotDataset(
-        const wxColor &dotColor,
-        const wxColor &dotStrokeColor,
-        wxVector<wxPoint2DDouble> &data,
-        const wxChartType &chartType=wxCHARTTYPE_LINE);
+    const wxColor &dotColor,
+    const wxColor &dotStrokeColor,
+    wxVector<wxPoint2DDouble> &data,
+    const wxChartType &chartType = wxCHARTTYPE_LINE, const bool &showDots = true);
```

Comment:
I think, it is better (more efficient) to pass `bool` by value instead of passing it as `const` reference. And the code will look shorter this way.